images-list

```diff
@@ -37,6 +41,7 @@ istio/proxyv2 rancher/istio-proxyv2 1.7.1
 istio/proxyv2 rancher/istio-proxyv2 1.7.3
 jettech/kube-webhook-certgen rancher/jettech-kube-webhook-certgen v1.2.1
 jettech/kube-webhook-certgen rancher/jettech-kube-webhook-certgen v1.5.0
+jimmidyson/configmap-reload rancher/jimmidyson-configmap-reload v0.2.2
```

Comment:
Is this new ? Weird that we are adding an older version.

Reply:
This already exists: https://hub.docker.com/r/rancher/jimmidyson-configmap-reload/tags?page=1&name=0.2

Reply:
It does already exist, but I noticed it wasn't there. Should we add it regardless in the event that we need to pull and retag it again?

Reply:
Why is this being updated? I don't see the image change in https://github.com/rancher/charts/pull/857/files

Reply:
My thought was that `images-list` was a source of truth in the event that we needed to repopulate the Rancher Docker registry. However, I spoke to Caleb and have since removed the image.